ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml

```diff
@@ -57,19 +57,19 @@ operator:
   bundles:
   - dockerfile_path: bundleci.Dockerfile
   substitutions:
-  - pullspec: quay.io/openshift/origin-sriov-dp-admission-controller:4.8
+  - pullspec: quay.io/openshift/origin-sriov-dp-admission-controller:4.9
     with: pipeline:sriov-dp-admission-controller
-  - pullspec: quay.io/openshift/origin-sriov-cni:4.8
+  - pullspec: quay.io/openshift/origin-sriov-cni:4.9
     with: sriov-cni
```

Comment:
@oribon just curious why we don't have `pipeline:sriov-cni` like other substitutions?

Reply:
don't know, added it and after inspecting the logs it seems that nothing changed - should we keep it this way to be the same as the other substitutions?

Reply:
@fedepaol do you know why? I see you added the initial config.

Reply:
Not sure I understand the question, they seem to be there https://github.com/openshift/release/blob/15aaf58c09fd14045e5b0c16bbd88c374fd44b88/ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml#L62

Reply:
> Not sure I understand the question, they seem to be there
> 
> https://github.com/openshift/release/blob/15aaf58c09fd14045e5b0c16bbd88c374fd44b88/ci-operator/config/openshift/sriov-network-operator/openshift-sriov-network-operator-master.yaml#L62

I noticed this PR added the pipeline prefix to sriov-cni.  I assume it was not there previously.

Reply:
ah now I get it sorry. I believe that was a mistake